### ISSUE TITLE
Unable to get messages older than 12 hours

### ISSUE THREAD

**Issue by n3xt-samuel** [svix/svix-webhooks#1455]
### Version
svix npm package v1.26.0

### Platform

N/A

### Description

I am using the svix package to get messages and noticed that I am unable to fetch messages older than 12 hours old (may be shorter).

I am able to query new messages.

Org name: n3xt
App Id: app_2mZbzzog8mEKxDMN3851uyMlym0

![Screenshot 2024-09-28 at 2 26 46 PM](https://github.com/user-attachments/assets/ac01ff5d-73c3-4771-b960-c501f865e3b2)

For example I am able to query for `fb3aa742-da1d-4749-8fd7-54256fc64ccc` but when I query for `5b99d7b3-5342-4da8-95c5-41a3b4d1514f`, I get this in the response 
```
Message: {"code":"not_found","detail":"message not found"}
```

Snippet
```
await svix.message.get("app_2mZbzzog8mEKxDMN3851uyMlym0", "fb3aa742-da1d-4749-8fd7-54256fc64ccc")); // Good
await svix.message.get("app_2mZbzzog8mEKxDMN3851uyMlym0", "5b99d7b3-5342-4da8-95c5-41a3b4d1514f"); // Not Found
```

### What I expect
I should be able to query older messages.

### What is happening
I am unable to query messages older than 12 hours.

**Comment by tasn** [comment]
Thanks for reporting, we'll take a look!

**Comment by svix-gabriel** [comment]
I took a look into this, but I haven't been able replicate the issue.

These are the steps I took to reproduce
1. Created a Message with an `eventId`
2. Called `GET /api/v1/app/:app_id/msg/:id` using the `eventId` from step 1. The message returned successfully.
3. A bit over 12h later, I called `GET /api/v1/app/:app_id/msg/:id`, again using the same `eventId`. The message returned successfully.

I also couldn't find anything in the code that would indicate the bug you described. Are the steps I took to reproduce accurate? Is there any other info about the message that might be relevant? (For example, did the message have a payload that expired or was manually deleted?).
